cord19-explorer/src/main/java/edu/asu/diging/cord19/explorer/config/SecurityContext.java

```diff
@@ -31,7 +31,8 @@ protected void configure(HttpSecurity http) throws Exception {
                 .and().authorizeRequests()
                 // Anyone can access the urls
                 .antMatchers("/", "/resources/**", "/login", "/loginFailed", "/register", "/location/**", "/paper/**",
-                        "/stats", "/affiliation/**", "/arxiv/**", "/logout", "/reset/**", "/search", "/affiliations","/publications")
+                        "/stats", "/affiliation/**", "/arxiv/**", "/logout", "/reset/**", "/search", "/affiliations",
+                        "/publications", "/pubs")
```

Comment:
same as before; please make sure to reopen existing pull requests

Reply:
I think I already made a comment about `/publications` and `/pubs` here? There should just be one